src/public/modules/forms/base/components/start-page.client.component.js

```diff
@@ -12,6 +12,7 @@ angular.module('forms').component('startPageComponent', {
     logoState: '@',
     formTitle: '@',
     authType: '<',
+    myInfoError: '<',
```

Comment:
lol the capitalisation here gives me anxiety

Reply:
ptsd